### ISSUE TITLE
Get `pactl` path dynamically

### ISSUE THREAD

**Issue by Ambrevar** [flexibeast/pulseaudio-control#7]
Currently pulseaudio.el fails on systems where `pactl` is not in `/usr/bin/pactl`.  (See NixOS, GuixSD.)  I know it's customizable, but it would be nice to make the default a bit smarter, as is customary in Emacs in general.  For instance:

```diff
-(defcustom pulseaudio-control-pactl-path "/usr/bin/pactl"
+(defcustom pulseaudio-control-pactl-path (executable-find "pactl")
```

**Comment by Ambrevar** [comment]
Any issue with my suggestion?

**Comment by flexibeast** [comment]
@Ambrevar Sorry to have taken so long to get to this. Thank you for the suggestion! This has now been implemented in c8640fce, and a MELPA package containing it has now been built. Can this issue be closed?

**Comment by Ambrevar** [comment]
Thanks!
